server/src/graphql/index.mjs

```diff
@@ -135,6 +135,7 @@ const typeDefs = gql`
     id: ID!
     from: SenderRecipient
     to: [SenderRecipient]!
+    cc: [SenderRecipient]
```

Comment:
```suggestion
    cc: [SenderRecipient]!
```

There will always be at least an empty array present

Reply:
done